src/clr-core/internal/base/focus-trap.base.spec.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { createTestElement, removeTestElement, waitForComponent, componentIsStable } from '@clr/core/test/utils';
+import { registerElementSafely } from '../utils/register.js';
+import { CdsBaseFocusTrap } from './focus-trap.base.js';
+
+// register the element just for testing purposes
+registerElementSafely('cds-base-focus-trap', CdsBaseFocusTrap);
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'cds-base-focus-trap': CdsBaseFocusTrap;
+  }
+}
+
+xdescribe('modal element', () => {
```

Comment:
Do we want to leave this xdescribe in? Our build will probably flag it...

Reply:
Ah I left it in by accident. Thanks for catching that.